mllib/src/main/scala/org/apache/spark/ml/evaluation/MulticlassClassificationEvaluator.scala

```diff
@@ -72,7 +73,39 @@ class MulticlassClassificationEvaluator @Since("1.5.0") (@Since("1.5.0") overrid
   @Since("3.0.0")
   def setWeightCol(value: String): this.type = set(weightCol, value)
 
-  setDefault(metricName -> "f1")
+  @Since("3.0.0")
+  final val metricClass: DoubleParam = new DoubleParam(this, "metricClass",
```

Comment:
@srowen I am somewhat regretful about the name. Maybe `metricLabel` is more suitable, since the metric names are `XXXByLabel`.  How about change them (multilabel+multiclass) to `metricLabel` in this PR?

Reply:
I'm OK with that too. Up to your preference.